source/documentors/references/doc_style_guide.rst

```diff
@@ -185,12 +185,11 @@ Images should be used to explain complex processes. They are a quick solution to
 
 To add an image to an Open edX document, use the following RST format, which has all the necessary parameters included:
 
-```rst
-.. image:: ../assets/example_title.jpg
-    :width: 500px
-    :align: center
-    :alt: This should describe the image for SEO and accessibility reasons.``
-
+.. code::
+    .. image:: ../assets/example_title.jpg
+        :width: 500px
+        :align: center
+        :alt: This should describe the image for SEO and accessibility reasons.
```

Comment:
.. code-block::

  .. image:: _assets/example_image_2.png
      :width: 500px
      :align: center
      :alt: Alternative text. this is important for accessibility